include/lmms_math.h

```diff
@@ -34,7 +34,7 @@
 #include <cmath>
 using namespace std;
 
-#if defined (LMMS_BUILD_WIN32) || defined (LMMS_BUILD_APPLE) || defined(LMMS_BUILD_HAIKU)  || defined (__FreeBSD__)
+#if defined (LMMS_BUILD_WIN32) || defined (LMMS_BUILD_APPLE) || defined(LMMS_BUILD_HAIKU)  || defined (__FreeBSD__) || defined(__OpenBSD__)
```

Comment:
Ok, I can't seem to find much reason for the discrepancies between compilers on these helpers but I still feel we shouldn't need platform-specific `#ifdef`'s.  Can we try removing the `#ifdefs`'s and just letting this block run on all platforms?